zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/ZeebeRestController.java

```diff
@@ -5,20 +5,19 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.zeebe.gateway.rest;
+package io.camunda.zeebe.gateway.rest.controller;
 
+import io.camunda.zeebe.gateway.rest.ConditionalOnRestGatewayEnabled;
 import java.lang.annotation.Documented;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE, ElementType.METHOD})
 @Documented
 @ConditionalOnRestGatewayEnabled
 @RestController
-@RequestMapping("/v1")
```

Comment:
:thought_balloon: Just commenting for clarity: we're removing this annotation from here and adding it explicitly to each controller since we will add `v2` of the REST API, and the Topology and UserTask endpoints need to support both versions.

:upside_down_face: I understand the reasoning, but it will be a bit annoying that we now have to remember to add this annotation to each controller and remember to annotate it in the future.

Reply:
Good point, yeah, I was also looking for alternatives but couldn't find anything nice yet 😄
Could we move to `"/v2"` for this annotation and only adjust the ones that exist for `v1` as well? 🤔

Reply:
I re-added "v2" path mapping to the annotation to ease the controller development. Controllers that don't change between "v1" and "v2" must override the mapping. Those will be rare anyway, I presume. The Topology controller might stay the same but all others most likely change.